features/bootup_animation.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Boot Animation
+
+    Scenario: The OS boots up to Overture
+        Given An OS which has not booted before
+         When The OS is booted
+         Then The boot animation displays
+          And The boot animation stops before overture
+
+    Scenario: The OS boots up to dashboard
+        Given An OS which has not booted before
```

Comment:
Should this be `An OS which has  booted before and gone through Overture` ?

Reply:
Yeah, definitely. I'll change it.